src/scripts.js

```diff
@@ -275,94 +190,236 @@ let updateTrendingStepDays = (user) => {
+  $('#stairs-user-stairs-today').text(activityData.find(activity => {
+    return activity.userID === user.id && activity.date === todayDate
+  }).flightsOfStairs * 12);
+  $('#steps-calendar-total-active-minutes-weekly').text(user.calculateAverageMinutesActiveThisWeek(todayDate));
 
-//
-// stairsTrendingButton.addEventListener('click', function() {
-//   user.findTrendingStairsDays();
-//   trendingStairsPhraseContainer.innerHTML = `<p class='trend-line'>${user.trendingStairsDays[0]}</p>`;
-// });
-//
-$('#steps-calendar-total-active-minutes-weekly').text(user.calculateAverageMinutesActiveThisWeek(todayDate));
+  $('#steps-calendar-total-steps-weekly').text(user.calculateAverageStepsThisWeek(todayDate))
 
-$('#steps-calendar-total-steps-weekly').text(user.calculateAverageStepsThisWeek(todayDate))
+  $('#steps-friend-active-minutes-average-today').text(userRepository.calculateAverageMinutesActive(todayDate));
 
-$('#steps-friend-active-minutes-average-today').text(userRepository.calculateAverageMinutesActive(todayDate));
+  $('#steps-friend-average-step-goal').text(`${userRepository.calculateAverageStepGoal()}`);
 
-$('#steps-friend-average-step-goal').text(`${userRepository.calculateAverageStepGoal()}`);
+  $('#steps-friend-steps-average-today').text(userRepository.calculateAverageSteps(todayDate));
 
-$('#steps-friend-steps-average-today').text(userRepository.calculateAverageSteps(todayDate));
+  $('#steps-info-active-minutes-today').text(activityData.find(activity => {
+    return activity.userID === user.id && activity.date === todayDate
+  }).minutesActive);
 
-$('#steps-info-active-minutes-today').text(activityData.find(activity => {
-  return activity.userID === user.id && activity.date === todayDate}).minutesActive);
+  $('#steps-user-steps-today').text(activityData.find(activity => {
+    return activity.userID === user.id && activity.date === todayDate
+  }).numSteps);
 
-$('#steps-user-steps-today').text(activityData.find(activity => {
-  return activity.userID === user.id && activity.date === todayDate}).numSteps);
+  let sortedHydrationDataByDate = user.ouncesRecord.sort((a, b) => {
+    console.log('oz', user.ouncesRecord);
+    if (Object.keys(a)[0] > Object.keys(b)[0]) {
+      return -1;
+    }
+    if (Object.keys(a)[0] < Object.keys(b)[0]) {
+      return 1;
+    }
+    return 0;
+  });
+
+  let weeklyHydrationDataArray = sortedHydrationDataByDate.splice(0, 7);
+  console.log('weekly', weeklyHydrationDataArray);
+  console.log('sorted', sortedHydrationDataByDate.splice(0, 7));
+  //Refactor this into a forEach
+  for (var i = 0; i < $('.daily-oz').length; i++) {
+    $('.daily-oz')[i].text(user.addDailyOunces(Object.keys(sortedHydrationDataByDate[i])[0]))
+  };
+  //splice sortedHydrationDataByDate to get most recent 7 entires
+  //loop through the first 7 entries and populate the oz per day
+}
 
-// user.findFriendsTotalStepsForWeek(userRepository.users, todayDate);
+// 1 WEEK LEFT CHECKLIST:
```

Comment:
Notes here for proceeding throughout the week. Please refer to this as we complete items and check them off as we go, and add any necessary tasks or to-dos.